charts/azure-k8s-metrics-adapter/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure Kubernetes Metrics Adapter
+
+An implementation of the Kubernetes [Custom Metrics API and External Metrics API](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis) for Azure Services. 
+
+This adapter enables you to scale your [application deployment pods](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) running on [AKS](https://docs.microsoft.com/en-us/azure/aks/) using the [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) (HPA) with metrics from Azure Resources (such as [Service Bus Queues](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues)) and custom metrics stored in Application Insights. Learn more about [using an HPA to autoscale with with external and custom metrics](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-metrics-not-related-to-kubernetes-objects).
+
+## Installing the Chart
+
+```sh
+helm install --name my-release charts/azure-k8s-metrics-adapter
+```
+
+## Uninstalling the Chart
+
+```sh
+helm delete my-release
+```
+
+## Configuration
+
+### All Available Configuration Values
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| `image.repository` | Image repository | `jsturtevant/azure-k8-metrics-adapter-amd64` |
```

Comment:
@jsturtevant Are these images going to reside under the `microsoft` repo Docker Hub?